mutant.sh

```diff
@@ -1,11 +1,12 @@
 #/usr/bin/bash -ex
 
-bundle exec mutant run                                           \
-  --zombie                                                       \
-  --ignore-subject Mutant::CLI#add_debug_options                 \
-  --ignore-subject Mutant::Isolation::Fork::Parent#call          \
-  --ignore-subject Mutant::Mutator::Node::Argument#skip?         \
-  --ignore-subject Mutant::Mutator::Node::ProcargZero#dispatch   \
-  --ignore-subject Mutant::Mutator::Node::When#mutate_conditions \
-  --ignore-subject Mutant::Zombifier#call                        \
+bundle exec mutant run                                                 \
+  --zombie                                                             \
+  --ignore-subject Mutant::CLI#add_debug_options                       \
+  --ignore-subject Mutant::Expression::Namespace::Recursive#initialize \
```

Comment:
A mutation for 3.0 specific code here. We need version specific configuration to exclude the subject (or mutation) on specific versions.